bambi/interpret/logs.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from bambi.interpret import logger
+
+
+def log_interpret_defaults(func):
+    interpret_logger = logger.get_logger()
```

Comment:
If I recall correctly, you need to pass `getLogger` a name to get the same logger. As it's done here https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/__init__.py#L30.

So you could do `logger.get_logger("bambi_interpret")`

Reply:
That's correct. When I did this, it wasn't returning the info logs to the console. Once I removed the name, it returned the info logs.